Celeste.Mod.mm/Mod/Meta/MapMeta.cs

```diff
@@ -132,9 +133,83 @@ public class MapMeta : IMeta {
+            if (!string.IsNullOrEmpty(TitleBaseColor)) { other.TitleBaseColor = TitleBaseColor; }
+            if (!string.IsNullOrEmpty(TitleAccentColor)) { other.TitleAccentColor = TitleAccentColor; }
+            if (!string.IsNullOrEmpty(TitleTextColor)) { other.TitleTextColor = TitleTextColor; }
+            if (IntroType != null) { other.IntroType = IntroType; }
+            if (Dreaming != null) { other.Dreaming = Dreaming; }
+            if (!string.IsNullOrEmpty(ColorGrade)) { other.ColorGrade = ColorGrade; }
+            if (!string.IsNullOrEmpty(Wipe)) { other.Wipe = Wipe; }
+            if (DarknessAlpha != null) { other.DarknessAlpha = DarknessAlpha; }
+            if (BloomBase != null) { other.BloomBase = BloomBase; }
+            if (BloomStrength != null) { other.BloomStrength = BloomStrength; }
+            if (!string.IsNullOrEmpty(Jumpthru)) { other.Jumpthru = Jumpthru; }
+            if (CoreMode != null) { other.CoreMode = CoreMode; }
+            if (!string.IsNullOrEmpty(CassetteNoteColor )) { other.CassetteNoteColor = CassetteNoteColor; }
+            if (!string.IsNullOrEmpty(CassetteSong )) { other.CassetteSong = CassetteSong; }
+            if (!string.IsNullOrEmpty(PostcardSoundID )) { other.PostcardSoundID = PostcardSoundID; }
+            if (!string.IsNullOrEmpty(ForegroundTiles )) { other.ForegroundTiles = ForegroundTiles; }
+            if (!string.IsNullOrEmpty(BackgroundTiles )) { other.BackgroundTiles = BackgroundTiles; }
+            if (!string.IsNullOrEmpty(AnimatedTiles )) { other.AnimatedTiles = AnimatedTiles; }
+            if (!string.IsNullOrEmpty(Sprites )) { other.Sprites = Sprites; }
+            if (!string.IsNullOrEmpty(Portraits )) { other.Portraits = Portraits; }
+            if (OverrideASideMeta != null) { other.OverrideASideMeta = OverrideASideMeta; }
+            if (Modes != null) {
+                if (other.Modes == null) {
+                    other.Modes = Modes;
+                } else {
+                    for (var i = 0; i < Modes.Length && i < other.Modes.Length; i++) {
+                        if (other.Modes[i] == null) {
+                            other.Modes[i] = Modes[i];
+                        } else if (Modes[i] != null) {
+                            Modes[i].AddTo(other.Modes[i]);
+                        } else {
+                        }
+                    }
+                }
+            }
+            if (Mountain != null) { other.Mountain = Mountain; }
+            if (CompleteScreen != null) { other.CompleteScreen = CompleteScreen; }
+            if (LoadingVignetteScreen != null) { other.LoadingVignetteScreen = LoadingVignetteScreen; }
+            if (LoadingVignetteText != null) { other.LoadingVignetteText = LoadingVignetteText; }
+            if (CassetteModifier != null) { other.CassetteModifier = CassetteModifier; }
+        }
+
+        public static MapMeta Add(MapMeta self, MapMeta other) {
```

Comment:
Add is like AddTo but it is static and works correctly if either of the metas are null.